### DISCUSSION TITLE
Hello! Is it possible to use this library to detect the distance between two points in an image?

### DISCUSSION THREAD

**Discussion by crapthings** [PeculiarVentures/GammaCV#184] [General]
Hello! Is it possible to use this library to detect the distance between two points in an image?
It's like a wooden plate where we put two points, which may be blue or red or green, that are distinct from the plate itself.

**Comment by WorldThirteen** [comment]
Hi, @crapthings! Sorry for the long response. GammaCV primarily gives you tools to manipulate image/numerical data using GPU power. You can build feature detection/tracking using the tools provided and then use some math to track the distance. It's not a trivial task though, since different cameras had different FOVs, etc.
GammaCV does not provide built-in capabilities to track the distance between points.
